mediapipe/__init__.py

```
@@ -11,3 +11,55 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from mediapipe.python import *
```

Ruff (0.11.9)

`from mediapipe.python import *` used; unable to detect undefined names

(F403)